tests/test_profiler.py

```diff
@@ -302,30 +302,42 @@ def test_pytorch_profiler_trainer_ddp(tmpdir, pytorch_profiler):
     """Ensure that the profiler can be given to the training and default step are properly recorded. """
     model = BoringModel()
     trainer = Trainer(
-        max_epochs=1,
         default_root_dir=tmpdir,
-        limit_train_batches=2,
-        limit_val_batches=2,
+        max_epochs=1,
+        limit_train_batches=5,
+        limit_val_batches=5,
         profiler=pytorch_profiler,
         accelerator="ddp",
         gpus=2,
     )
     trainer.fit(model)
 
-    expected = ('validation_step', 'training_step_and_backward', 'training_step', 'backward')
+    if _KINETO_AVAILABLE:
+        expected = ('validation_step', )
+    else:
+        expected = ('validation_step', 'training_step_and_backward', 'training_step', 'backward')
     for name in expected:
-        assert sum(e.name == name for e in pytorch_profiler.function_events)
+        assert sum(e.name == name for e in pytorch_profiler.function_events), name
 
     files = set(os.listdir(pytorch_profiler.dirpath))
     expected = f"fit-profiler-{trainer.local_rank}.txt"
     assert expected in files
 
-    path = os.path.join(pytorch_profiler.dirpath, expected)
-    assert Path(path).read_text()
+    path = pytorch_profiler.dirpath / expected
+    assert path.read_text("utf-8")
+
+    if _KINETO_AVAILABLE:
```

Comment:
Could we have a separate test regarding KINETO? Would help to know when the tests are skipped and easier to debug.

Reply:
I think it's fine considering we are testing the same thing. It's just not present before 1.8.1 so that's why we have the if